content/blog/async-pipe-is-not-pure/index.md

```diff
@@ -0,0 +1,346 @@
+---
+{
+	title: "Async pipe is not pure 🤯",
+	description: "Learn how angular's async pipe is not pure and how that's not a problem, by building async pipe from scratch.",
+	published: '2023-01-18T18:00:00.284Z',
+	authors: ['eneajahollari'],
+	tags: ['angular', 'performance', 'async'],
```

Comment:
We generally try to limit our tags to only match the main topics on the site thus far. Let's change this to:

```suggestion
tags: ['angular'],
```